dockerfiles/gleam-1.10.Dockerfile

```
@@ -1,12 +1,15 @@
 # syntax=docker/dockerfile:1.7-labs
-FROM ghcr.io/gleam-lang/gleam:v1.9.1-erlang-alpine
+FROM ghcr.io/gleam-lang/gleam:v1.10.0-node-slim
 
 ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="gleam.toml,manifest.toml"
 
 WORKDIR /app
 COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)